output/csv/config.go

```diff
@@ -94,10 +104,7 @@ func ParseArg(arg string, logger *logrus.Logger) (Config, error) {
 		case "fileName":
 			c.FileName = null.StringFrom(r[1])
 		case "timeFormat":
-			c.TimeFormat = TimeFormat(r[1])
-			if !c.TimeFormat.IsValid() {
```

Comment:
I'd suggest checking if `c.TimeFormat.IsValid()`, and if it's not, write a warning log with a message that was initially an error.